lib/autoproj/ops/main_config_switcher.rb

```diff
@@ -16,16 +16,21 @@ def initialize(ws)
             EXPECTED_ROOT_ENTRIES = [".", "..", "autoproj_bootstrap",
                                      ".autoproj", "bootstrap.sh", ENV_FILENAME].to_set
 
+
+            def interactive_flag_from_env
```

Comment:
would expect this to be "ignore_empty_dir_from_env"  while interactive_flag_from_env should refer to AUTOPROJ_NONINTERACTIVE

Reply:
I didn't recall the existence of `AUTOPROJ_NONINTERACTIVE`. I'd deprecate AUTOPROJ_BOOTSTRAP_IGNORE_NONEMPTY_DIR in favor of `NONINTERACTIVE` to be honest. Fine with you if I test for both ?

Reply:
Sure

Reply:
In the end I followed your original idea, and kept IGNORE_NONEMPTY. You just have NONINTERACTIVE == '1' that implies IGNORE_NONEMPTY